sdk/core/src/CommonNetworkMember/BaseNetworkMember.ts

```diff
@@ -709,6 +709,10 @@ export abstract class BaseNetworkMember {
     return this.signUnsignedCredential(unsignedCredential, keyType)
   }
 
+  async validateCredential(signedCredential: any) {
```

Comment:
is type `VCNamePersonV1` will be correct from `import { VCNamePersonV1 } from '@affinidi/vc-data' ?